projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html

```diff
@@ -97,6 +97,66 @@ <h1>{{ record.metadata.name }}</h1>
         </ng-container>
       </dl>
     </tab>
+    <tab>
+      <ng-template tabHeading>
+        <i class="fa fa-university mr-2"></i>{{ 'Acquisition settings' | translate }}
+      </ng-template>
+      <!-- ACQUISITION SETTINGS -->
+      <dl class="row" *ngIf="record.metadata.acquisition_settings as acquisitionSettings">
+        <ng-container *ngIf="acquisitionSettings.shipping_informations as acqInfos">
+          <dt class="col-3" translate>Shipping informations :</dt>
+          <div class="col-9 card container py-2 mb-2">
+            <div class="row">
+              <div class="col-6 d-flex">
+                <i class="fa fa-address-card-o pr-2 pt-1"></i>
+                <div>
+                  {{ acqInfos.name }}<br/>
+                  <ng-container *ngIf="acqInfos.address">
+                    <ng-container *ngIf="acqInfos.address.street">{{ acqInfos.address.street }}<br/></ng-container>
```

Comment:
Why not using a `div` or `ul li` instead of `br`?

Reply:
I will force a new line, despite CSS or any other stuff. In this case the `br` tag seems the easier and useful.